docs/data-model.md

```diff
@@ -20,36 +20,74 @@ nav = [
+Rows and columns can represent anything (they could even represent the same set of things - a [bigraph](https://en.wikipedia.org/wiki/Bigraph)). Pilosa can associate arbitrary key/value pairs (referred to as attributes) to rows and columns, but queries and storage are optimized around the core matrix.
 
-Pilosa lays out data first in rows, so queries which get all the set bits in one or many rows, or compute a combining operation on multiple rows such as Intersect or Union are the fastest. Pilosa also has the ability to categorize rows into different "frames" and quickly retrieve the top rows in a frame sorted by the number of bits set in each row.
+Pilosa lays out data first in rows, so queries which get all the set bits in one or many rows, or compute a combining operation on multiple rows such as Intersect or Union are the fastest. Pilosa categorizes rows into different *frames* and quickly retrieve the top rows in a frame sorted by the number of bits set in each row.
 
-Please note that Pilosa is most performant when row and column IDs are sequential starting from 0. You can deviate from this to some degree, but if you try to set a bit with column ID 2^63, bad things will start to happen.
+Please note that Pilosa is most performant when row and column IDs are sequential starting from 0. You can deviate from this to some degree, but setting a bit with column ID 2<sup>63</sup> on a single-node cluster, for example, will not work well due to memory limitations.
 
 ![basic data model diagram](/img/docs/data-model.svg)
 *Basic data model diagram*
 
 ### Index
 
-The purpose of the Index is to represent a data namespace. You cannot perform cross-index queries.  Column-level attributes are global to the Index.
+The purpose of the Index is to represent a data namespace. You cannot perform cross-index queries.
 
 ### Column
 
-Column ids are sequential increasing integers and are common to all Frames within an Index.
+Column ids are sequential increasing integers and are common to all Frames within an Index. A single column often corresponds to a record in a relational table, although other configurations are possible, and sometimes preferable (see genomics TODO).
 
 ### Row
 
 Row ids are sequential increasing integers namespaced to each Frame within an Index.
 
 ### Frame
 
-Frames are used to segment and define different functional characteristics within your entire index.  You can think of a Frame as a table-like data partition within your Index.
+Frames are used to segment rows within an index, for example to defining different functional groups. A frame might correspond to a single field in a relational table, where each row in a standard frame represents a single possible value of the field. Similarly, a frame with BSI values could represent all possible integer values of a field .
```

Comment:
just noting that "index" is lowercase here, but capitalized in other places. 
"to defining" > "to define"

Reply:
We aren't very consistent with styling Pilosa technical terms. In the white paper some are lowercase and italicized, others are wrapped in quotation marks. In the docs we have the option to link to a relevant section (glossary for example). We don't have any existing guidelines for this, do we?